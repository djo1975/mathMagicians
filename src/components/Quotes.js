import React from 'react';
import Header from './Header';

const Quotes = () => (
  <div>
    <Header />
    <div className="quotes">
      <p>Mathematics is not about numbers, equations, computations,</p>
      <p>or algorithms: it is about understending. -William Paul Thurston</p>
    </div>
  </div>
);

export default Quotes;
