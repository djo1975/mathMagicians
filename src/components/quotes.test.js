import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from '@cfaester/enzyme-adapter-react-18';
import Quotes from './Quotes';

Enzyme.configure({ adapter: new Adapter() });

describe('Quotes', () => {
  it('should renders correctly', () => {
    const component = shallow(<Quotes />);
    expect(component).toMatchSnapshot();
  });
});
