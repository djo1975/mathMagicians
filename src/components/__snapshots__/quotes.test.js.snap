// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes should renders correctly 1`] = `ShallowWrapper {}`;
